roq-plugin/asciidoc-common/deployment/src/main/java/io/quarkiverse/roq/plugin/asciidoc/common/deployment/AsciidocHeaderParser.java

```diff
@@ -58,12 +60,16 @@ public static JsonObject toPageData(Header header) {
         pageData.put("attributes", JsonObject.mapFrom(attributes));
         for (String key : attributes.keySet()) {
             if (key.startsWith("page-")) {
-                pageData.put(key.substring(5), attributes.get(key));
+                final String value = attributes.get(key);
+                pageData.put(key.substring(5), value.isEmpty() ? true : value);
             }
         }
         if (!header.title().isBlank()) {
             pageData.put(Page.FM_TITLE, header.title());
         }
+        if (header.attributes().containsKey(Page.FM_DESCRIPTION) && !header.attributes().get(Page.FM_DESCRIPTION).isBlank()) {
```

Comment:
How do we make sure asciidoc attribute eventual description is not overriden by FM description.

If we don't we should then document the precedence.

Reply:
The FM parser has priority over the Asciidoc parser
Also, here, if asciidoc headers contains `description`, we add it to the data (but it won't override the FM)